daily-usage/ldm-daily-usage.py

```diff
@@ -233,15 +233,20 @@ def main():
     parser.add_argument('--date', action='store', help='Limit display to single date, must be in format 2021-11-24. If the date is not available then there will be a blank entry.')
     args = parser.parse_args()
 
-    with open(args.config, 'r') as f:
-        config = json.load(f)
-
-    if args.debug:
-        HTTPConnection.debuglevel = 1
-        config['debug'] = True
-        logging.basicConfig(level=logging.DEBUG)
-
-    return daily_usage(config, args)
+    try:
+        with open(args.config, 'r') as f:
+            config = json.load(f)
+            
+        if args.debug:
+            HTTPConnection.debuglevel = 1
+            config['debug'] = True
+            logging.basicConfig(level=logging.DEBUG)
+
+        return daily_usage(config, args)
```

Comment:
Any exception from daily_usage function will be treated as a JSONDecodeErrror. The try-except should just be around the 

`config = json.load(f)`